Website/src/items/templates/edgewebsite/Site Config.yml

```diff
@@ -0,0 +1,40 @@
+﻿---
+ID: "d55741f0-482d-4001-9230-f8ffc391f06a"
+Parent: "59c21fbf-d36e-5462-9847-c8c0bf7786ab"
+Template: "ab86861a-6030-46c5-b394-e8f99e8b87db"
+Path: /sitecore/templates/Project/edgewebsite/Site Config
```

Comment:
In https://github.com/Sitecore/Sitecore.Demo.Edge/pull/91 , @scottmulligan introduced the `/sitecore/templates/Project/edgewebsite/Data/Site Configuration` template for the same goal (storing the global website page title). He told me you two had discussed it and choose to store that information directly in Sitecore instead of it being pushed by Content Hub.

So I guess this template, and the code of the processor in here are outdated and should be updated to use Scott's new template when that PR will be merged to develop. Is that the plan?

Reply:
didnt see that template in /sitecore/templates/Project/edgewebsite/Data/Site Configuration
so keeping the one i created

Reply:
This template is definitely in #91 :
![image](https://user-images.githubusercontent.com/2000200/132692328-675170f0-f193-4d43-b8d6-cacb825deedf.png)